src/style.css

```diff
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-family: Avenir, Arial, sans-serif;
+}
+
+.form {
+  position: relative;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  margin-top: 20px;
+}
+
+.input {
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  width: 100%;
+}
+
+.first__input {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+}
+
+.first__input::placeholder {
+  font-family: Avenir, Arial, sans-serif;
+  font-size: 16px;
+  font-weight: 300px;
+  color: #3d4e61;
+}
+
+#big__icon {
```

Comment:
use class instead id

Reply:
check it everywhere

Reply:
it is unique item, why can`t it be styled by id? there is no more big items to use class for